accounts/tests/views/test_user.py

```diff
@@ -10,6 +11,10 @@ class UserTestCase(APITestCase):
     def setUp(self):
         self.user = User.objects.create(email="user@staff.com")
 
+    def test_user_detail_url_fail(self):
+        with self.assertRaises(NoReverseMatch):
+            reverse("user-detail")
+
```

Comment:
so there's no need for testing if PUT, PATCH, DELETE are not allowed, since they're under the `user-detail` URL name?

Reply:
yep, the url doesn't get created if we're not using any of the detail action mixins! so it'd be impossible for those to work